### PR TITLE
Copy gcapi 1049

### PR DESCRIPTION
**Pull request by VWIJECH**
Latest Typography changers.

### CODE REVIEW

**Comment by umahaea** [comment]
@uparavi , @VWIJECH - can we wait until we get the elements V0 branch with your dev code changes? bcos, if we think of merging into ux-test-platform master, then it would break the master.. I guess the elements V0 branch is still not ready with your dev code changes

File: src/main/java/elements/elementsPageObjects/TypographyPageObjects.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	
+	
+	
+	public By heading_2 = By.id("hd_22");
+	public By heading_3 = By.id("hd_33");
+	public By heading_4 = By.id("hd_44");
+	public By heading_5 = By.id("hd_55");
+	public By heading_6 = By.id("hd_66");
+	
+
```
Comment (umahaea) [inline src/main/java/elements/elementsPageObjects/TypographyPageObjects.java]: can we cut down these unwanted line spaces?

File: src/test/java/elementsTests/TypographyTest.java [hunk 2]
```diff
@@ -0,0 +1,1110 @@
...
+						"rgba(35, 31, 32, 1)" },
+				{ typoPgObj.unorderedListChildItem1, "16px", "22px",
+						"rgba(35, 31, 32, 1)" },
+				{ typoPgObj.unorderedListGrandChildItem1, "16px", "22px",
+						"rgba(35, 31, 32, 1)" }, };
+	}
+
+	// Test 2
+	@Test(testName = "Lists font fest", dataProvider = "getListsFontTestData", groups = { "desktop" })
+	private void ListsFontTest(By element, String fontsize, String lineheight,
```
Comment (umahaea) [inline src/test/java/elementsTests/TypographyTest.java]: I would like the see better formatting in the code. It looks a bit shaky, especially in the @dataprovider methods. Some have closing braces in the same line. Closing '}' braces should be adjacent to the method name, please refer to one of my other methods.

**Comment by VWIJECH** [comment]
We completed test automation for Code,Body,Header,List .
Following files were newly added.
Typography.java
Typography.html
TypograghyPageObjects.java
All formatting changers also committed.

**Pull request closed without merging**